Hackney.Shared.Finance.Tests/Boundary/Response/ResponseWrapperTests.cs

```diff
@@ -0,0 +1,54 @@
+﻿using Xunit;
+using FluentAssertions;
+using Hackney.Shared.Finance.Boundary.Response;
+
+namespace Hackney.Shared.Finance.Tests.Boundary.Response
+{
+    public class ResponseWrapperTests
+    {
+        public ResponseWrapperTests()
+        { }
+
+        [Fact]
+        public void Given_A_Response_Entity_WhenSingle_Then_ResposeWrapper_ShouldNotBeEmpty()
+        {
+            var response = new DummyResponse();
+            var wrapperResponse = new ResponseWrapper<DummyResponse>(response);
+
+            wrapperResponse.IsEmpty.Should().BeFalse();
+        }
+
+
+        [Fact]
+        public void Given_A_Response_Entity_WhenNull_Then_ResposeWrapper_ShouldBeEmpty()
```

Comment:
Method naming convention on the tests within this file.
I believe dotnet throws exceptions for: "you shouldn't use underscores within method names". At least, it does so within the API and listener projects.
Was this error suppressed, or do we not have 'Treat warnings as errors" enabled?